test/testdata/packager/layered/business1/__package.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+# packager-layers: utility,power,business,api,service
+
+class Business1 < PackageSpec
+  strict_dependencies 'layered'
+  layer 'business'
+
+  import Service1 # error: `Business1` is at layer `business`, so it can not import package `Service1`, which is at layer `service`
```

Comment:
Do you think it's worth putting the phrase "layering violation" in the error message? It's the thing that we use to document all of this and if it were present, would be very easy to search for when someone encounters it

```
Layering violation: attempted to import `Service1` (`layer 'service'`) from `Business1` (`layer 'business'`)
```

or something like that.

Reply:
I agree that the phrase "layering violation" would be helpful to include (and I guess "strict dependencies" violation for the strict deps errors).

How about:
```
Layering violation: cannot import `Service1` (in `layer 'service'`) from `Business1` (in `layer 'business'`)
```
I think the "cannot import" is important to convey what the problem is.